src/braket/experimental/algorithms/qc_qmc/classical_qmc.py

```diff
@@ -0,0 +1,367 @@
+import copy
+import multiprocessing as mp
+import os
+from dataclasses import dataclass
+from typing import Callable, List, Tuple
+
+import pennylane as qml
+from openfermion.circuits.low_rank import low_rank_two_body_decomposition
+from pennylane import numpy as np
+from scipy.linalg import det, expm, qr
+
+
+@dataclass
+class ChemicalProperties:
+    h1e: np.ndarray  # one-body term
+    eri: np.ndarray  # two-body term
+    nuclear_repulsion: float  # nuclear repulsion energy
+    v_0: np.ndarray  # one-body term stored as np.ndarray with mean-field subtraction
+    h_chem: np.ndarray  # one-body term stored as np.ndarray, without mean-field subtraction
+    v_gamma: List[np.ndarray]  # 1j * L_gamma
+    L_gamma: List[np.ndarray]  # Cholesky vector decomposed from two-body terms
```

Comment:
I would love to see a style guide or some kind of source encouraging breaking code style for established scientific names. For now, I'm not convinced that it's worth the deviation